content/jest.setupFilesAfterEnv.ts

```diff
@@ -0,0 +1,18 @@
+import * as logger from '@well-known-components/logger'
+
+beforeAll(() => {
+  // Mock logger implementation
+  const createLogComponent = logger.createConsoleLogComponent
+  jest.spyOn(logger, 'createLogComponent').mockImplementation(() => {
+    const logComponentMock = createLogComponent()
+    const loggerMock = logComponentMock.getLogger('__test__')
+    loggerMock.debug = () => {}
+    loggerMock.error = () => {}
+    loggerMock.info = () => {}
+    loggerMock.log = () => {}
+    loggerMock.warn = () => {}
+    logComponentMock.getLogger = jest.fn(() => loggerMock)
+    return logComponentMock
```

Comment:
This is not ultimately necessary, but currently our CI logs are bloated with unnecessary data, making them hard to read like [here](https://app.circleci.com/pipelines/github/decentraland/catalyst/4484/workflows/50038e87-2322-43fa-9c2a-e5dd4d294ab2/jobs/24434). I believe this could be mitigated in the past by setting the `LOG_LEVEL` env variable, but I understand the current logger implementation is lacking that capability.
So, the point of these changes is to improve readability of the tests by mocking the current logger.

What would you suggest in this case?

Reply:
I'd sacrifice readability in the console runner to benefit debug capabilities. Now that every test is passing ` bloated with unnecessary data` is a valid argument, but when you are trying to fix a hairy bug then you really need those logs.

I'd make it optional with an env var to enable local debugging with logs or add the log level env var to here https://github.com/well-known-components/logger/blob/2a1765ec3a3b5236c691f91a6083a9c5d1745694/src/helpers.ts#L12 if that suffices your use case